### DISCUSSION TITLE
Mod crashes when playing background matchmaking

### DISCUSSION THREAD

**Discussion by marokw** [jugeeya/UltimateTrainingModpack#652] [Q&A]
Please do something with mod crashes wthat occures when I play background matchmaking. I really like playing like this because connecting with next player can take some time in my country. It would be great if I could use yours Training Mod with Background Matchmaking but it crashes when I train in it and new rival is found. Unfortunately I don't know which features of this mod cause crashes. Maybe you could add "Online Safe Mode" version of this Mod or something like that.

**Comment by austintraver** [comment]
As best I understand, this mod shouldn't cause any interference with Background Matchmaking. Are you using any other mods?

**Reply by marokw** [reply]
I use many skins, smashline and arc-network. So I understand that on your switch Background Matchmaking works fine with training mod?

**Reply by marokw** [reply]
Error is: SSBU Training Modpack has panicked at 'called 'Option::unwrap()' on a 'None' value', src/common/mod.rs:54:41
